File: web-app/src/app/services/types/Jobstudentstatus.ts
import { JobpostingsEligiblestudent } from "./JobpostingsEligibleStudent";

export type Jobstudentstatus = {
  Id: number;
  Name?: string;
  JobpostingsEligiblestudents?: JobpostingsEligiblestudent[];
};

export type PostJobstudentstatus = {
  JobpostingsEligiblestudents?: {
    StudentId?: number;
    JobPostingId?: number;
    StatusId?: number;
  }[];
};

File: Placements.WebApi/Placement/Models/Jobstudentstatus.cs
using System;
using System.Collections.Generic;

namespace Placements.WebApi.Placement.Models;

public partial class Jobstudentstatus
{
    public int Id { get; set; }

    public string? Name { get; set; }

    public virtual ICollection<JobpostingsEligiblestudent> JobpostingsEligiblestudents { get; set; } = new List<JobpostingsEligiblestudent>();
}
